seed/seed20.go

```diff
@@ -429,7 +429,25 @@ type errorComponentNotInSeed struct {
 	error
 }
 
-func (s *seed20) lookupVerifiedComponent(cref naming.ComponentRef, snapRev snap.Revision, snapID, snapProvenance string, snapsDir string, handler ContainerHandler, tm timings.Measurer) (Component, error) {
+func modelContainsComponent(model *asserts.Model, cref naming.ComponentRef) bool {
+	for _, sn := range model.AllSnaps() {
```

Comment:
again it feels like maybe we should compute an exploded map early in LoadAssertions ?

Reply:
I wonder if this is a good case for a member variable for the struct (initialized in `LoadAssertions`)? Otherwise we'd be passing it around quite a lot to be able to share it for multiple snaps.

Reply:
yes, I was thinking of a field in the struct